keras_cv/layers/regularization/dropblock_2d.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from keras.engine.base_layer import BaseRandomLayer
+from keras.utils import conv_utils
```

Comment:
Same comment here - just copy paste the utility into a `keras_cv.utils.conv_utils` file for now.  just the single one you need.  it is ok to duplicate some code for now.

Reply:
Unless import tf.keras.utils.conv_utils works

Reply:
Added `conv_utils.py` file and this function to it.